tools/importer/README.md

```diff
@@ -0,0 +1,50 @@
+## Porting various repositories into mbed-os
+
+importer.py script can be used to import code base from different repositories into mbed-os.
+
+### Pre-requisties
+1. Get the `required` repository clone and update it to commit required. Note: Repository should be placed outside the mbed-os.
+2. Create json file as per template
+
+### JSON file template
+
+You can list all the files and folders which are required to be copied in config file in `file` and `folder` arrays respectively. Script will remove the files/folders specified in `src` section from mbed-os repository before copy operation is performed.
+New commit is created on branch `feature_+repo_name+last_sha` with commit message "[REPO_NAME]: Updated to last_sha"
+
+Note: Only files present in folder will be copied, directories inside the folder will not be copied.
+
+`commit_sha` is list of commits present in mbed-os repo. These commits will be applied after copying files and folders listed above. Each commit is cherry-picked and applied sequentially in list with `-x` option, to record SHA of otriginal commit.
+Note: In case of conflict, you will have to resolve the conflict and make sure appended statement "(cherry picked from commit …​)" is not deleted from commit message. Re-execute the python script to apply rest of the SHA commits.
+
+{
+  "files" : [
+    {
+      "src_file" : "CMSIS/Core/Template/ARMv8-M/tz_context.c",
+      "dest_file" : "cmsis/TARGET_CORTEX_M/mbed_tz_context.c"
+    },
+    ...
+    {
+      "src_file" : "",
+      "dest_file" : ""
+    }
+  ],
+  "folders" : [
+    {
+      "src_folder" : "CMSIS/Core/Include/",
+      "src_folder" : "cmsis/TARGET_CORTEX_M/"
```

Comment:
This is now allowed in JSON or python dictionaries. Perhaps you meant to have "dest_folder" here?